internal/todo/todo.go

```diff
@@ -19,8 +20,8 @@ import (
 const name = "todo"
 
 // doc defines the help text for the todo linter.
-const doc = `Ensures that each TODO comment defined in the codebase conforms to one of the
-following formats: TODO(<gh-user>)[<jira-ticket>]: <summary> or TODO[<jira-ticket>]: <summary>`
+const doc = "Ensures that each TODO comment defined in the codebase conforms to the " +
+	"format `TODO(<gh-user>)[<jira-ticket>]: <summary>`, with one of `(gh-user)` or `[jira-ticket]` being required."
```

Comment:
I like this change, but I am concerned about introducing this all of a sudden.

Can we communicate externally that we're going to turn this on before merging this?